modules/output/xml/main.nf

```diff
@@ -8,24 +8,24 @@ process WRITE_XML {
     executor 'local'
 
     input:
-    val matchesFiles  // {query prot seq md5: {model acc: match}}
-    val outputPath
-    val seqDbPath
+    val matches_files  // {query prot seq md5: {model acc: match}}
+    val output_prefix
+    val seq_db_file
     val nucleic
-    val ips6Version
+    val interproscan_version
+    val interpro_version
 
     exec:
     def writer = new StringWriter()
     def xml = new MarkupBuilder(writer)
     // set the correct encoding so symbols are formatted correctly in the final output
     xml.setEscapeAttributes(false) // Prevent escaping attributes
     xml.setEscapeText(false)       // Prevent escaping text
```

Comment:
AI hallucination? I think the only thing this line does is adding the following at the top of the XML output:

```xml
<setEscapeText>false<setEscapeText>
```

Reply:
It was needed to fix formatting of strings (typically description string), for example `&` was written as `&amp` and quotes within strings as `&quot` or `&apos`, but we aren't seeing those as I hadn't realised InterPro entries, go terms and pathways are still listed as to do in the XML writer. I'll add those now.